index.js

```diff
@@ -2,21 +2,111 @@
  * Reverse Polish Notation Calculator
  */
 
-var arguments = process.argv.slice(2);
+let arguments = process.argv.slice(2);
 
-var operand1 = arguments[0];
-var operand2 = arguments[1];
-var operator = arguments[2];
+let operator = [];
+let operand = [];
 
-console.log('Calculating:', operand1, operand2, operator);
+for (let k = 2; k < arguments.length; k++) {
+  if (k % 2 == 0) {
+    operator.push(arguments[k]);
+  }
+}
+
+for(let i = 0; i < arguments.length; i++){
+  if (i % 2 == 0){
+
+  }else{
+    operand.push(arguments[i]);
+  }
+
+  if(i==0){
+    operand.push(arguments[i]);
+  }
+}
+
+if(arguments == 0){
+  console.log('Supported operators:\n\r +, -, *, /');
+  console.log();
+  console.log('Usage example:\n\r 5 5 + 9 *');
+}
+
+if(operator == 0){
+  console.log("No valid operator!");
+  console.log('Supported operators:\n\r +, -, *, /');
+  console.log();
+  console.log('Usage example:\n\r 5 5 + 9 *');
+}
 
-var result;
+if(operand.length == operator.length){
+  console.log('Too many operators passed!');
+}
+
+if(operand.length > operator.length+1){
+  console.log('Too less operators passed!');
+}
```

Comment:
Added failsave for malformed arguments entered while usage of [index.js ](https://github.com/dwdab/ws20-devops-calc/blob/main/index.js)